### PR TITLE
Security announcement

### PR DESCRIPTION
**Pull request by unak**
Please do not publish before merging release announcement.

### CODE REVIEW

File: en/news/_posts/2018-03-28-poisoned-nul-byte-dir-cve-2018-8780.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+layout: news_post
+title: "CVE-2018-8779: Unintentional directory traversal by poisoned NUL byte in Dir"
```
Comment (nagachika) [inline en/news/_posts/2018-03-28-poisoned-nul-byte-dir-cve-2018-8780.md]: CVE-2018-8780

File: en/news/_posts/2018-03-28-poisoned-nul-byte-dir-cve-2018-8780.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+author: "usa"
+date: 2018-03-28 14:00:00 +0000
+tags: security
+lang: en
+---
+
+There is an unintentional directory traversal in some methods in `Dir`.
+This vulnerability has been assigned the CVE identifier [CVE-2018-8779](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-8779).
```
Comment (nagachika) [inline en/news/_posts/2018-03-28-poisoned-nul-byte-dir-cve-2018-8780.md]: /8779/8780/

File: en/news/_posts/2018-03-28-poisoned-nul-byte-unixsocket-cve-2018-8779.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+---
+layout: news_post
+title: "CVE-2018-8780: Unintentional socket creation by poisoned NUL byte in UNIXServer and UNIXSocket"
```
Comment (nagachika) [inline en/news/_posts/2018-03-28-poisoned-nul-byte-unixsocket-cve-2018-8779.md]: /8780/8779/

File: en/news/_posts/2018-03-28-poisoned-nul-byte-unixsocket-cve-2018-8779.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+author: "usa"
+date: 2018-03-28 14:00:00 +0000
+tags: security
+lang: en
+---
+
+There is a unintentional socket creation vulnerability in `UNIXServer.open` method of socket library bundled with Ruby.
+And there is also a unintentional socket access vulnerability in `UNIXSocket.open` method.
+This vulnerability has been assigned the CVE identifier [CVE-2018-8780](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-8780).
```
Comment (nagachika) [inline en/news/_posts/2018-03-28-poisoned-nul-byte-unixsocket-cve-2018-8779.md]: /8780/8779/

File: en/news/_posts/2018-03-28-unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+So, if a script accepts an external input as the prefix, and the targeted directory has inappropriate permissions or the ruby process has inappropriate privileges, the attacker can create a directory or a file at any directory.
+
+All users running an affected release should upgrade immediately.
+
+## Affected Versions
+
+* Ruby 2.2 series: 2.2.9 and earlier
+* Ruby 2.3 series: 2.3.6 and earlier
+* Ruby 2.4 series: 2.4.3 and earlier
+* Ruby 2.5 series: 2.5.0 and earlier
```
Comment (marocchino) [inline en/news/_posts/2018-03-28-unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914.md]: @unak Is this `and earlier` intentional?
It is not match with japanese one.

Comment (unak) [reply en/news/_posts/2018-03-28-unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914.md]: Oh, I didn't take care of the nuance.
Of course, Ruby 2.5 series had only one release 2.5.0, so we don't have to mention about "earlier".
But, about thinking preview versions, "earlier" might have some meaning.
Conclusion: you can choose to say or not to say "earlier" as you like :)

Comment (marocchino) [reply en/news/_posts/2018-03-28-unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914.md]: Thanks for your reply :)

**Pull request merged**